pkg/daemon/ceph/osd/device.go

```diff
@@ -311,13 +311,14 @@ func getMetadataPartitionDetails(cfg *osdConfig) (*config.PerfSchemePartitionDet
 		return nil, fmt.Errorf("partition scheme missing from %+v", cfg)
 	}
 
-	metadataPartitionType := cfg.partitionScheme.GetMetadataPartitionType()
-
+	// If Filestore, there is no seperate metadata partition. Will return before looking it up.
```

Comment:
i can't find any callers of the `getMetadataPartitionDetails` method. Should we remove this whole function too?

Reply:
Won't this be something we might need when we place the WAL and DB on seperate devices? i thought we might need it then and hence left it in there.

Reply:
@jbw976 do you remember the history of this? Is it a surprise that this method isn't being called? I just worry about bit rot if it's dead code until and if the WAL and DB separation is added later.

Reply:
@travisn @jbw976 cleaned out all the entities as discussed. Thanks for the clarifications.